### ISSUE TITLE
Exposing message from Lua

### ISSUE THREAD

**Issue by technomancy** [syohex/emacs-lua#1]
This is fantastic; thanks. It's impressive how you can already do a lot with a little.

However, it's somewhat limiting since as far as I can tell, `print` doesn't work from within the Lua side. It would be a lot easier to debug issues if the Lua side had access to the emacs-lisp `message` function. Would it make sense to add this?

**Comment by syohex** [comment]
https://github.com/syohex/emacs-lua/commit/0010e5b2077796fe9b4eedaf50148ed32b362581

I'm not sure, what you mean is like this ?

```elisp
(let ((ls (lua-new)))
  (lua-do-string ls "                                                                                                                                                                                                                  
message(\"%s %d\", \"Hello Lua\", 42)                                                                                                                                                                                                  
"))
```

**Comment by technomancy** [comment]
That's exactly what I was looking for; thanks!

**Comment by syohex** [comment]
I see. I'll merge it later

**Comment by syohex** [comment]
I have merged #2
